README.rst

```diff
@@ -69,6 +69,15 @@ You can specify the base URL using a `configuration file`_:
   [pytest]
   base_url = http://www.example.com
 
+Using an environment variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can specify the base URL with an environment variable:
+
+.. code-block:: bash
+
+   export PYTEST_BASE_URL="http://www.example.com"
```

Comment:
I usually try to avoid giving examples of how to set environment variables. This is because there are several ways you can do it, and it varies significantly between platforms. Personally I would just document something like "You can specify the base URL by setting the `PYTEST_BASE_URL` environment variable."